cardano-api/test/Test/Cardano/Api/Typed/CBOR.hs

```diff
@@ -191,4 +186,37 @@ roundtrip_Tx_Cddl (AnyCardanoEra era) =
 -- -----------------------------------------------------------------------------
 
 tests :: TestTree
-tests = $testGroupGenerator
+tests = testGroup "Test.Cardano.Api.Typed.CBOR"
+  [ testPropertyNamed "roundtrip witness byron CBOR"                         "roundtrip witness byron CBOR"                         prop_roundtrip_witness_byron_CBOR
```

Comment:
Can we just avoid all together using `testPropertyNamed`? It seems like a lot of additional trouble for not much benefit. Hedgehog will display the property name by default.